[Java11] com.ibm.ws.microprofile.metrics_fat_tck: com.ibm.ws.microprofile.metrics.tck.launcher.MetricsTCKLauncher.launchTck

RTC 258571

```
junit.framework.AssertionFailedError: 2018-10-02-21:36:46:059 No TCK test JUnit result files were found in the results directory which suggests the TCK tests did not run - check build logs.
ResultsDir: /home/jazz_build/_KvHIIMZyEeiA6ZPkBDHUqw-EBC.PROD.WASRTC-BSA-000-00-00/jbe/build/dev/com.ibm.ws.microprofile.metrics_fat_tck/autoFVT/results/tck/surefire-reports
at componenttest.topology.utils.MvnUtils.findJunitResultFiles(MvnUtils.java:411)
at componenttest.topology.utils.MvnUtils.postProcessTestResults(MvnUtils.java:369)
at componenttest.topology.utils.MvnUtils.runTCKMvnCmd(MvnUtils.java:343)
at com.ibm.ws.microprofile.metrics.tck.launcher.MetricsTCKLauncher.launchTck(MetricsTCKLauncher.java:60)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at componenttest.custom.junit.runner.FATRunner$1.evaluate(FATRunner.java:193)
at componenttest.custom.junit.runner.FATRunner$2.evaluate(FATRunner.java:323)
at componenttest.custom.junit.runner.FATRunner.run(FATRunner.java:167)
```

The following was found in mvnOutput_TCK
```
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] MicroProfile Metrics TCK Runner 1.0 TCK Module ..... FAILURE [08:02 min]
[INFO] MicroProfile Metrics TCK 1.0 Runner ................ SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 08:06 min
[INFO] Finished at: 2018-10-14T06:54:23Z
[INFO] Final Memory: 20M/35M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project tck.runner.tck: Could not resolve dependencies for project com.ibm.ws.microprofile.metrics:tck.runner.tck:jar:1.0-SNAPSHOT: Could not find artifact com.sun:tools:jar:1.6.0 at specified path /home/jazz_build/_XhJOEM9jEeio4_O0Fzon5g-EBC.PROD.WASRTC-9LP-000-00-00/jbe/jvms/openj9_jdk11_20180929-sdk/../lib/tools.jar -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException

```

--- a/dev/com.ibm.ws.microprofile.metrics_fat_tck/publish/tckRunner/tck/pom.xml
+++ b/dev/com.ibm.ws.microprofile.metrics_fat_tck/publish/tckRunner/tck/pom.xml
@@ -16,7 +16,7 @@
 	<name>MicroProfile Metrics TCK Runner 1.0 TCK Module</name>
 
 	<properties>
-		<microprofile.metrics.version>1.0.1</microprofile.metrics.version>
+		<microprofile.metrics.version>1.0.2-RC2</microprofile.metrics.version>
 		<arquillian.version>1.1.13.Final</arquillian.version>
 
 		<!-- the below is used in arquillian.xml -->
@@ -72,12 +72,18 @@
 			<version>1.2</version>
 			<scope>provided</scope>
 		</dependency>
-		<dependency>
+		<!--<dependency>
 			<groupId>com.ibm.ws.org.jboss.arquillian.container</groupId>
 			<artifactId>arquillian-wlp-managed-8.5</artifactId>
 			<version>1.0.0.Final-SNAPSHOT</version>
 			<scope>test</scope>
-		</dependency>
+		</dependency>-->
+        <dependency>
+            <groupId>io.openliberty.arquillian</groupId>
+            <artifactId>arquillian-liberty-managed</artifactId>
+            <version>1.0.3</version>
+            <scope>test</scope>
+        </dependency>
 		<dependency>
 			<groupId>javax.inject</groupId>
 			<artifactId>javax.inject</artifactId>
@@ -131,10 +137,10 @@
                         <dependency>org.eclipse.microprofile.metrics:microprofile-metrics-api-tck</dependency>
                     </dependenciesToScan>
                     <excludes>
-                       <exclude>**/TimerTest.java</exclude>
+                       <!-- <exclude>**/TimerTest.java</exclude>
                        <exclude>**/MpMetricTest.java</exclude>
                        <exclude>**/HistogramTest.java</exclude>
-                       <exclude>**/MeterTest.java</exclude>
+                       <exclude>**/MeterTest.java</exclude> -->
                     </excludes>
                </configuration>
 			</plugin>

--- a/dev/com.ibm.ws.microprofile.metrics_fat_tck/publish/tckRunner/tck/pom.xml
+++ b/dev/com.ibm.ws.microprofile.metrics_fat_tck/publish/tckRunner/tck/pom.xml
@@ -72,12 +72,6 @@
 			<version>1.2</version>
 			<scope>provided</scope>
 		</dependency>
-		<!--<dependency>
-			<groupId>com.ibm.ws.org.jboss.arquillian.container</groupId>
-			<artifactId>arquillian-wlp-managed-8.5</artifactId>
-			<version>1.0.0.Final-SNAPSHOT</version>
-			<scope>test</scope>
-		</dependency>-->
         <dependency>
             <groupId>io.openliberty.arquillian</groupId>
             <artifactId>arquillian-liberty-managed</artifactId>
@@ -137,10 +131,10 @@
                         <dependency>org.eclipse.microprofile.metrics:microprofile-metrics-api-tck</dependency>
                     </dependenciesToScan>
                     <excludes>
-                       <!-- <exclude>**/TimerTest.java</exclude>
+                       <exclude>**/TimerTest.java</exclude>
                        <exclude>**/MpMetricTest.java</exclude>
                        <exclude>**/HistogramTest.java</exclude>
-                       <exclude>**/MeterTest.java</exclude> -->
+                       <exclude>**/MeterTest.java</exclude>
                     </excludes>
                </configuration>
 			</plugin>

